jsapp/js/projects/projectsTable/projectActions.module.scss

```diff
@@ -5,16 +5,19 @@
 .root {
   @include mixins.centerRowFlex;
   gap: sizes.$x10;
-}
 
-.menu {
-  @include mixins.floatingRoundedBox;
-  padding: sizes.$x6;
-  min-width: sizes.$x180;
+  // Override .k-button[disabled] styles to allow tooltips on disabled buttons
```

Comment:
> 1. every button get's some `span.buttonWrapper`
> 2. we put `data-tooltip` on `.buttonWrappers`
> 3. we stop overriding button styles

👍 yeah, I should have thought of this.

In some places this is how tooltip components work generally – [Chakra Tooltip](https://chakra-ui.com/docs/components/tooltip/usage)

> Alternatively we can modify the Button component to stop using pointer-events and thus allowing tooltips to work on disabled button. We would need to add some :not([disabled]) for some selectors (like hover styles or cursor) and prevent the onClick callback. This might be a better solution I think and more useful for codebase :)

I'd like is. I figured it'd be better saved for a new PR, though.

The onClick callback does [seem to respect `[disabled]`](https://stackoverflow.com/questions/20294867/how-to-trigger-a-click-event-on-disabled-elements) even if you don't override pointer-events, so we might not have to do that ourselves.

Reply:
If we go with the `.buttonWrapper`, and then we introduce the Button disabled state improvement - it would mean we should remove those wrappers

Reply:
I'm going with wrapper, it's not hard to remove. (and it would continue to function even if a button upgrade happened underneath it.)

Substantial button upgrades would need to deal with both issues:

- The `:not([disabled])` pointer events overrides like you mentioned for hovers or cursors
- In our design system, disabled buttons are styled with `opacity: 0.5`. All DOM descendents including tooltips can never be more than 50% opaque after that, so tooltips would need a wrapper anyway or do css hacks case-per-case (possible here because of single icon buttons, but not doable in general without special color mixing.) I don't see a good way forward with this, other than to normalize tooltip wrappers, or redesign disabled buttons.

Reply:
One thing a future `<Tooltip>` wrapper could do is make it easier to add ARIA attributes that match the intended labels.

Ref:
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label, 

Speaking of tooltips, these would be on my wishlist:
- Multi-line tooltips
- Automatically avoid being cropped by the screen edge (so you can use center even on the edge), and wrap long tooltips on narrow screens

Reply:
Made a GH issue for Button improvement: https://github.com/kobotoolbox/kpi/issues/4611

Reply:
Made a GH issue for Tooltip improvement: https://github.com/kobotoolbox/kpi/issues/4612